website/pages/index.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import Head from 'next/head';
+import Navigation from '../components/navigation/Navigation';
 
 export default function Home() {
-  return <div>Hello, world!</div>;
+  return (
```

Comment:
do you mind adding a basic test for the page? it could just be a Jest snapshot to start off. this would mostly be an exercise to flesh out how the folder structure will look, how jest needs to be configured with nextjs, etc.

Reply:
Yep no problem